osu.Game/Skinning/LegacyJudgementPieceOld.cs

```diff
@@ -46,15 +44,6 @@ public virtual void PlayAnimation()
             const double fade_out_length = 600;
 
             this.FadeInFromZero(fade_in_length);
-            this.Delay(fade_out_delay).FadeOut(fade_out_length);
-
-            // legacy judgements don't play any transforms if they are an animation.... UNLESS they are the temporary displayed judgement from new piece.
```

Comment:
I'm not sure removing this is correct? Yes the "missed tick" portion of this is no longer a thing, but the early-return should still be here. It dates back to 2020 and https://github.com/ppy/osu/pull/10878.

Reply:
This shouldn't be removed, no. I was just testing it in passing, is an accidental commit.

Reply:
Should be fixed.